app/_locales/en/messages.json

```diff
@@ -344,6 +344,20 @@
   "addressCopied": {
     "message": "Address copied!"
   },
+  "addressMismatch": {
+    "message": "Site address mismatch"
+  },
+  "addressMismatchOriginal": {
+    "message": "Current URL: $1",
+    "description": "$1 replaced by origin URL in confirmation request"
+  },
+  "addressMismatchPunycode": {
+    "message": "Punycode version: $1",
+    "description": "$1 replaced by punycode version of the URL in confirmation request"
+  },
+  "addressMismatchWarning": {
```

Comment:
Minor, but for consistency the existing alerts have been using `alertMessageX` and `alertReasonX` for the translation keys.

Reply:
done 👍